### DISCUSSION TITLE
Feature: Export frames as tiles

### DISCUSSION THREAD

**Discussion by Lilith-In-Starlight** [Orama-Interactive/Pixelorama#571] [Ideas]
It should let you choose where you want each frame to be when saving it and remember where each frame is located, so that you don't have to set it up everytime you export.

It should come with an addition to the Tile Mode, to allow the user to select what tiles to show around the current frame, this to help the artist make sure a set of tiles connect correctly with each other.

**Comment by OverloadedOrama** [comment]
In v0.7 the export dialog window had been overhauled. It lets you preview your frames in the form of a spritesheet and change its orientation and the number of its columns or rows. You can also include frames of specific tags in your spritesheet. I guess that this can also be used to export tilesets. Is this good enough, or are you suggesting something else?
![image](https://user-images.githubusercontent.com/35376950/85629293-44842e00-b67a-11ea-9f10-398116aa2478.png)
